src/components/Categories/EditCategory.vue

```diff
@@ -1,78 +1,101 @@
 <template>
-        <div class="container" :style="{marginLeft:'30px'}">
-			<form >
-				<div class="row">
-					<div class="col-md-8 col-md-offset-2">
-						<h3  id="padding-page">Chỉnh sửa thể loại</h3>
-                            <div class="form-group">
-                                    <label for="name">Mã thể loại<span class="require">*</span></label>
-                                    <input type="text" name="id"  id="padding-bottom-border" :value="category.id"  class="form-control" readonly >
-                            </div>
-                             <div class="form-group">
-								<label for="name">Tên thể loại<span class="require">*</span></label>
-								<input type="text" name="name"  id="padding-bottom-border" :value="category.name"  class="form-control" >
-							</div>
-							<div class="form-group">
-								<label for="description">Mô tả<span class="require">*</span></label>
-								<input name="description" :value="category.description"  id="padding-bottom-border" class="form-control" />
-							</div>
-							<div class="form-group">
-								<input type = "submit" class="btn btn-primary" target="__blank" value="Cập nhật" @click="routeToEdit(author.id)"
-								:style="{borderRadius: '10px'}" />
-								<input type = "submit" class="btn default" target="__blank" value="Hủy" @click="goBack()" 
-								:style="{marginLeft:'10px', border: '1px solid black', borderRadius:'10px'}" />
-							</div> 
-                    </div>
-                </div>
-			</form>
-		</div>
-
+  <div class="container" :style="{ marginLeft: '30px' }">
+    <form>
+      <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+          <h3 id="padding-page">Chỉnh sửa thể loại</h3>
+          <div class="form-group" :style="{ marginTop: '40px' }">
+            <span class="p-float-label">
+              <InputText id="id" type="text" v-model="category.id" disabled />
+              <label for="id" :style="{ fontSize: '15px' }">Mã thể loại</label>
+            </span>
+          </div>
+          <div class="form-group" :style="{ marginTop: '40px' }">
+            <span class="p-float-label">
+              <InputText id="name" type="text" v-model="category.name" />
+              <label for="name" :style="{ fontSize: '15px' }"
```

Comment:
Muốn chữ to thì dùng h2, h3... nhưng hạn chế. 
Muốn highlight cái gì đó thì không nên làm nó to ra.

Thay vì thế, cứ đặt trong thẻ <strong>

Reply:
dạ em hiểu rồi ạ